src/testrender/cuda/wrapper.cu

```diff
@@ -131,7 +130,10 @@ process_closure(const OSL::ClosureColor* closure_tree)
             const char* mem = (const char*)((OSL::ClosureComponent*)cur)->data();
             const char* dist_str = *(const char**)&mem[0];
 
-            if (HDSTR(dist_str) == STRING_PARAMS(default))
+            OSL::ustring dist_u(dist_str);
```

Comment:
Build error with OptiX enabled:
```
cuda/wrapper.cu(133): error: calling a __host__ function("OpenImageIO_v2_6_0::ustring::ustring(const char *)") from a __device__ function("process_closure") is not allowed
```

Reply:
@steenax86, I think that dist_str should already be a ustringhash already because it was previously just being cast to HDSTR:
`if (HDSTR(dist_str) == STRING_PARAMS(default))`
So you should just be able to change line 131 from
`const char* dist_str = *(const char**)&mem[0];`
to
 `OSL::ustringhash dist_uh = *(OSL::ustringhash*)&mem[0];`
and then get ride of lines 133 and 134.